tests/testcases/040_check-network-adv.yml

```diff
@@ -0,0 +1,43 @@
+---
+- hosts: all
+  tasks:
+
+    - name: Test tunl0 routes
+      shell: "! ip ro | grep '/26 via' | grep -v tunl0"
+      when: ipip|default(false)
+
+- hosts: kube-master
+  tasks:
+
+    - name: Get services
+      shell: kubectl get svc --namespace=default
+      register: get_svc
+      run_once: true
+
+    - name: Deploy netchecker
+      shell: "curl -s https://raw.githubusercontent.com/bogdando/netchecker-server/dev/utils/deploy_netchecker.sh | bash"
```

Comment:
but that's not ``curl | sudo bash`` at least! :)
It is used for CI only, then destroyed with the env. But I agree would be nice to make it looking nicer, eventually